### ISSUE TITLE
Formatting example in the Docs does not work

### ISSUE THREAD

**Issue by mrvaita** [andialbrecht/sqlparse#694]
Hello,

I was just playing around with this package and I saw that the default [example](https://sqlparse.readthedocs.io/en/latest/intro/#getting-started) for the SQL format function does not work as expected.

Running the following code:
```
import sqlparse

sql = 'select * from foo where id in (select id from bar);'
print(sqlparse.format(sql, reindent=True, keyword_case='upper'))
```
gives the output
```
SELECT *
FROM foo
WHERE id in
  (SELECT id
   FROM bar);
```

expected output:
```
SELECT *
FROM foo
WHERE id IN
  (SELECT id
   FROM bar);
```

I used sqlparse version 0.4.3 and python3.9

I think this is due to the fact that `IN` is interpreted as a comparison operator and not as a keyword.

Thanks a lot everyone for the time.

**Comment by andialbrecht** [comment]
Agreed. The behavior changed in `0.4.0`. Before that release `in` was formatted uppercase in that example.

**Comment by andialbrecht** [comment]
Fixed in e9241945801808d1db7f76bdccbbe9a200042c37
